### PR TITLE
fix: validate preview cli

### PR DESCRIPTION
**Pull request by rephus**
Closes: #9088

### Description:

Without --preview 
Screenshot from 2024-03-05 11-05-34

with --preview 

image

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/cli/src/index.ts [hunk 1]
```diff
@@ -525,6 +525,7 @@ program
     .option('--vars <vars>')
     .option('--threads <number>')
     .option('--no-version-check')
+    .option('--preview', 'Validate the last preview if available', false)
```
Comment (rephus) [inline packages/cli/src/index.ts:528]: for now I'm only doing this on `validate` and not on `deploy` because I?m not sure if that will break some existing deploys for users . we can do it on a separate PR

**Review by ZeRego — APPROVED** [review]
